Document/0x05b-Basic-Security_Testing.md

```diff
@@ -440,12 +440,18 @@ rdr pass inet proto tcp from any to any port 5236 -> 127.0.0.1 port 8080
 " | sudo pfctl -ef -
 ```
 
-- The interception proxy must listen to the port specified in the port forwarding rule above (port 8080).
+**Intercepting the Requests**
+
+The interception proxy must listen to the port specified in the port forwarding rule above (port 8080).
 
 Start the app and trigger a function that uses FCM. You should see HTTP messages in your interception proxy.
 
 ![Intercepted Messages](Images/Chapters/0x05b/FCM_Intercept.png)
 
+**End-to-End Encryption for Push Notifications**
+
+As an additional layer of security, push notifications can be encrypted by using [Capillary](https://github.com/google/capillary "Capillary"). Capillary is a library to simplify the sending of end-to-end (E2E) encrypted push messages from Java-based application servers to Android clients. 
+
```

Comment:
Can you please add references at the end?

Reply:
For this section we always use the links as part of the text and didn't have a reference table at the end. I can create an additional table at the end with tools used in this section. Makes also sense for the printed book to see the URLs.

Reply:
Maybe it is best to have links here and references as well?

Reply:
Done :-)